test/parallel/test-runner-coverage.js

```diff
@@ -428,3 +428,72 @@ test('coverage with included and excluded files', skipIfNoInspector, () => {
   assert.strictEqual(result.status, 0);
   assert(!findCoverageFileForPid(result.pid));
 });
+
```

Comment:
I think we may want to create a new `test/parallel/test-runner-coverage-thresholds.js` file. Sometimes when these test files start to get big, they can time out on the slower CI machines.